### ISSUE TITLE
Repeatedly enabling/disabling camera crashes the app.

### ISSUE THREAD

**Issue by Tamulur** [EnoxSoftware/HoloLensWithOpenCVForUnityExample#20]
I need to enable/disable marker detection repeatedly in my app. Doing this in this example app (switch between Aruco example and main menu back and forth) crashes after a while, unless I build the app in Debug configuration. How to safely disable/enable marker tracking repeatedly?

**Comment by EnoxSoftware** [comment]
Thank you very much for reporting.
Does this crash only occur when running ArUcoExample?

**Comment by Tamulur** [comment]
So far yes. I tried Comic filter, face detection and Aruco calibration, and they haven't crashed for me yet.

**Comment by Abdul-Mukit** [comment]
I think this is the reason. VulcanTechnologies/HoloLensCameraStream#29
I faced a similar sort of problem. Repeatedly stopping/starting the camera overtime caused higher RAM usage until eventually, the app would crash. For my application, I eventually put the camera helper and dlib script on a prefab. Every time I needed to stop the camera I would destroy the GO with the camera script. When I want to resume service, I create another instance of that prefab. It solved the memory issue for me. For my application, I don't need this start/stop too often so the small delay during initializing the new instance was not a big deal for me.
